implementation/jobberknoll/Dockerfile

```diff
@@ -1,13 +1,24 @@
-FROM denoland/deno:2.1.4
-
-WORKDIR /app
+FROM ubuntu:24.10 AS runner-base
 
 RUN apt-get -y update
-RUN apt-get -y install curl
+RUN apt-get -y install curl libargon2-1
```

Comment:
As discussed in PM's, this leaves litter in images. I'll optimize this out soon™